server/src/main/java/dev/keva/server/command/impl/string/Get.java

```diff
@@ -15,15 +16,23 @@
 @ParamLength(1)
 public class Get {
     private final KevaDatabase database;
+    private final ExpirationManager expirationManager;
 
     @Autowired
-    public Get(KevaDatabase database) {
+    public Get(KevaDatabase database, ExpirationManager expirationManager) {
         this.database = database;
+        this.expirationManager = expirationManager;
     }
 
     @Execute
     public Reply<?> execute(byte[] key) {
-        val got = database.get(key);
-        return got == null ? BulkReply.NIL_REPLY : new BulkReply(got);
+        boolean expirable = expirationManager.isExpirable(key);
```

Comment:
should we validate for set as well? like key is not allowed to have internal postfix.

Reply:
postfix has a special character, so it's unlikely to be accidentally set